### PR TITLE
Lets the players know that they can steal just the photo of the blueprints instead of the entire thing.

### PR DESCRIPTION
**Pull request by tipoima**
## What Does This PR Do
Theft objectives that accept a photo of an item (currently only being the blueprints) now state that outright

## Why It's Good For The Game
As far as I know, the only place mentioning this is the "High-Risk Items" page on the wiki. It's good to explain a mechanic in-game.

## Testing
Turned myself into a traitor until I rolled blueprints theft. Saw the new message. Did not see the message on any other objective.

## Changelog
:cl:
tweak: Blueprints theft objective now explains that a photo of the blueprints also counts.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -592,7 +592,9 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 /datum/objective/steal/update_explanation_text()
 	explanation_text = "Steal [steal_target.name]. One was last seen in [get_location()]. "
 	if(length(steal_target.protected_jobs) && steal_target.job_possession)
-		explanation_text += "It may also be in the possession of the [english_list(steal_target.protected_jobs, and_text = " or ")]."
+		explanation_text += "It may also be in the possession of the [english_list(steal_target.protected_jobs, and_text = " or ")]. "
+	if(/obj/item/photo in steal_target.altitems)
```
Comment (Contrabang) [inline code/game/gamemodes/objective.dm:596]: I'd rather you make this a var on steal_target, something like
```suggestion
	explanation_text += steal_target.extra_information
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Spaghetti-bit — APPROVED** [review]

**Comment by S34NW** [comment]
Hey there, although this is technically not a fix or refactor and thus should fall under the freeze rules, we have decided to permit it. This is due to the fact the freeze has only just begun and the size of this change.
Please familiarise yourself with the freeze rules for the coming month before making another PR. ❤️

**Comment by tipoima** [comment]
> Hey there, although this is technically not a fix or refactor and thus should fall under the freeze rules, we have decided to permit it. This is due to the fact the freeze has only just begun and the size of this change. Please familiarise yourself with the freeze rules for the coming month before making another PR. ❤️

I've seen the announcement, it's just that I wasn't sure if it would qualify as a fix or an addition; and people on discord said it's fine. I suppose I should've just waited instead.

File: code/game/gamemodes/steal_items.dm [hunk 2]
```diff
@@ -84,6 +86,7 @@
 	protected_jobs = list("Chief Engineer")
 	altitems = list(/obj/item/photo)
 	location_override = "the Chief Engineer's Office"
+	extra_information = "Stealing a photograph of the item is also an option."
```
Comment (Fordoxia) [inline code/game/gamemodes/steal_items.dm:89]:
```suggestion
	extra_information = "Obtaining a photograph of the blueprints is also an option."
```
Minor wording change + explicitly saying it's the blueprints you want to photograph.

**Review by Fordoxia — CHANGES_REQUESTED** [review]

**Review by Contrabang — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**